src/pages/Services.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { RenderImage } from "../components/Services";
+
+
+
+function App() {
```

Comment:
So we want to avoid calling all our components App.

This component holds the "RenderImage" component but it doesn't look like we actually need this. We can condense the logic of this component by placing it in the "RenderImage" component.